swift/ql/src/queries/Security/CWE-312/CleartextLogging.ql

```diff
@@ -0,0 +1,24 @@
+/**
+ * @name Clear-text logging of sensitive information
```

Comment:
One further comment, apologies...

Can we change this to something like "Good practices for logging sensitive information"? Alternatively, is there a way we can avoid using the stacked noun modifier `Clear-text` with the hyphen?

Reply:
As most parts of the docs in this query, this has been carried over from the version of this query in Go, JavaScript, Python, and Ruby. Also, see the `@id` of this query (`clear-text-logging`) which can't be easily changed.

Consistency with other languages aside, I have nothing against renaming this to `Cleartext logging of sensitive information`, or something like `Insertion of sensitive information into log files` (like in Java), but unless you have a strong opinion about it, I'd prefer not using `Good practices for logging sensitive information` since it doesn't seem to indicate the actual problem being highlighted, i.e. the problem is rather the lack of good practices (?)

Reply:
Makes sense, thank you for the explanation! I'd suggest `Cleartext logging of sensitive information` in that case.

Reply:
```suggestion
 * @name Cleartext logging of sensitive information
```